### PR TITLE
[zh] Translate eventratelimit API (v1alpha1)

### PR DESCRIPTION
**Pull request by tengqm**
related: #34221

### CODE REVIEW

File: content/zh-cn/docs/reference/config-api/apiserver-eventratelimit.v1alpha1.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+（`string` 类型的别名）
+
+**出现在：**
+
+- [Limit](#eventratelimit-admission-k8s-io-v1alpha1-Limit)
+
+<!--
+<p>LimitType is the type of the limit (e.g., per-namespace)</p>
+-->
+<p>LimitType 是限制类型（例如，per-namespace）。</p>
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/config-api/apiserver-eventratelimit.v1alpha1.md:151]: Maybe we can use `例如：per-namespace` or `例如 per-namespace`

File: content/zh-cn/docs/reference/config-api/apiserver-eventratelimit.v1alpha1.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
+---
+title: Event Rate Limit Configuration (v1alpha1)
+content_type: tool-reference
+package: eventratelimit.admission.k8s.io/v1alpha1
+auto_generated: true
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/config-api/apiserver-eventratelimit.v1alpha1.md:5]: Hey, you missed this. 😝
Localized pages are not auto generated.

File: content/zh-cn/docs/reference/config-api/apiserver-eventratelimit.v1alpha1.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
+---
+title: Event Rate Limit Configuration (v1alpha1)
```
Comment (Sea-n) [inline content/zh-cn/docs/reference/config-api/apiserver-eventratelimit.v1alpha1.md:2]: Not sure should this be translated or not.

**Review by howieyuen — APPROVED** [review]
/approve

**Pull request merged**